src/photosui.cs

```diff
@@ -525,6 +529,10 @@ interface PHPickerConfiguration : NSCopying
 		[Export ("preferredAssetRepresentationMode", ArgumentSemantic.Assign)]
 		PHPickerConfigurationAssetRepresentationMode PreferredAssetRepresentationMode { get; set; }
 
+		[NoWatch, NoTV, NoMac, iOS (15, 0), MacCatalyst (15,0)]
```

Comment:
Lets me consistent with the spaces, either (15, 0) or (15,0)

Reply:
I changed that and also removed the NoWatch, NoTV and NoMac since the interface header already had those labels

Reply:
Oops sorry, actually forgot it haha